client/lib/impure-lodash/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import {
```

Comment:
My thought was mostly around discoverability in case someone needed to add a test.

For example there's nothing stopping someone from still using `import { debounce } from 'lodash'` correct?

If they add a test later, we'd need to update usage to `import { debounce } from 'lib/impure-lodash';`